app/views/brexit_landing_page/_buckets.html.erb

```diff
@@ -1,11 +1,16 @@
 <div class="landing-page__section"
   data-analytics-ecommerce
   data-ecommerce-start-index="1"
-  data-list-title="Brexit landing page: <%= t('brexit_landing_page.browse') %>"
+  data-list-title="Brexit landing page: <%= t('brexit_landing_page.guidance_header') %>"
   data-search-query>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h2 class="govuk-heading-m landing-page__section-title"><%= t('brexit_landing_page.browse') %></h2>
+      <h2 class="govuk-heading-m landing-page__section-title">
+        <%= t('brexit_landing_page.guidance_header') %>
+      </h2>
+      <p>
+        <%= t('brexit_landing_page.guidance_subheader') %>
```

Comment:
In that case, can we make this:

```
<p><%= t('brexit_landing_page.guidance_subheader') %></p>
```

rather than indenting the text?

Reply:
what about the class? makes it a very long line?
doesn't get rendered inline does it?

Reply:
Just from a code point of view, I think it makes it easier to read. If there were a lot of attributes on the paragraph, I would do what you're doing here, but as there aren't it should be readable on one line.

Reply:
this might be outdated. this p has a class now:       

```
<p class="landing-page__guidance_subheader">
   <%= t('brexit_landing_page.guidance_subheader') %>
</p>
```

if i put it on one line it's a long one, no?